### DISCUSSION TITLE
Hi Can you build

### DISCUSSION THREAD

**Discussion by YYUUUU303** [AndreRH/hangover#137] [Q&A]
I am very much a noob, and in need paid support. I have a very small .exe and needs to run on my M1 Macbook. It is a authentication app for secure log into to a database that only used Microsoft edge (OMG)..can you help? I cant opperate in the termanial enviroment..and UTM emulation of a full X86 windows install is just too slow and edge is always crashing..thanks in advance. 

I can install this exe via windows on arm via Paralells, but there is driver issue and it wont work. 

It seems so close to work with Wine or maybe Hangover..but beyond me..

The app is 8mb

I have M1 Macbook pro

Their operation enviroment - (Japanese company=

- Windows 11／Windows 10
*Compatible with 32-bit (x86)/64-bit (x64) environments.
* Internet Explorer launched from the Windows 8.1 Start screen (Modern UI) is not supported (including support). Please use it with the conventional Windows desktop.
*Windows RT is not supported (including support).
*Some models do not have JAVA functions as standard.
If the JAVA function is not installed, it will not work properly, so please install the JAVA function in advance.
Intel Core2 Duo or later (1.2 GHz or higher) or AMD CPU with equivalent performance
* ARM-made or ARM-based CPUs cannot be used.
* Microsoft Edge
Microsoft Edge - Internet Explorer mode

Their developers say the ARM suport is a long time away/// they only moved from Internet Explorer only 1 year ago

**Comment by AndreRH** [comment]
I suggest to try a virtual Ubuntu Installation first. Getting hangover up and running inside it should be much easier as native Mac has the issue with the page size. See also https://github.com/AndreRH/hangover/issues/130 and https://github.com/AndreRH/hangover/issues/97

**Comment by YYUUUU303** [comment]
Ubuntu ARM64 installed via Parallels..what next? Perhaps we could take this
private? Where can I pm you please.

**Reply by AndreRH** [reply]
Now simply follow the steps from the README.md
If you need a private chat, there's Patreon
